examples/resources/spectrocloud_workspace/resource.tf

```diff
@@ -0,0 +1,80 @@
+data "spectrocloud_cluster" cluster1 {
+  name = "vsphere-picard-2"
+}
+
+resource "spectrocloud_workspace" "workspace" {
+  name = "wsp-tf"
+
+  clusters {
+    name = data.spectrocloud_cluster.cluster1.name
+    uid = data.spectrocloud_cluster.cluster1.id
+  }
+
+  cluster_rbac_binding {
+    type = "ClusterRoleBinding"
+
+    role = {
+      kind = "ClusterRole"
+      name = "testrole3"
+    }
+    subjects {
+      type = "User"
+      name = "testRoleUser4"
+    }
+    subjects {
+      type = "Group"
+      name = "testRoleGroup4"
+    }
+    subjects {
+      type      = "ServiceAccount"
+      name      = "testrolesubject3"
+      namespace = "testrolenamespace"
+    }
+  }
+
+  cluster_rbac_binding {
+    type      = "RoleBinding"
+    namespace = "test5ns"
+    role = {
+      kind = "Role"
+      name = "testrolefromns3"
+    }
+    subjects {
+      type = "User"
+      name = "testUserRoleFromNS3"
+    }
+    subjects {
+      type = "Group"
+      name = "testGroupFromNS3"
+    }
+    subjects {
+      type      = "ServiceAccount"
+      name      = "testrolesubject3"
+      namespace = "testrolenamespace"
+    }
+  }
+
+  namespaces {
+    name = "test5ns"
```

Comment:
Just a heads up for `IsNamespaceRegex` bool will be added into the api in 2.7 itself

Reply:
Adding support for regex